controller/haproxy/server.go

```diff
@@ -91,3 +91,20 @@ func (s *Server) UpdateServerSsl(value string) error {
 	}
 	return nil
 }
+
+func (s *Server) UpdateGrpc(value string) error {
+	enabled, err := utils.GetBoolValue(value, "enable-grpc")
+	if err != nil {
+		return err
+	}
+	if enabled {
+		s.Ssl = ""
+		s.Verify = "none"
+		s.Proto = "h2"
+	} else {
```

Comment:
Unnecessary `else` here, just use `return nil`

Reply:
else is necessary, and this approach is conflicting with 'server-ssl' annotation,
what happens when both of them are enabled ?

looking at the code, this will always override what  'server-ssl'  do, and even if this is false it will still override what server-ssl does because this has a default value